jsx/Form.js

```diff
@@ -314,9 +315,10 @@ class SelectElement extends Component {
     let options = e.target.options;
 
     // Multiple values
-    if (this.props.multiple && options.length > 1) {
+    if (this.props.multiple && numOfOptions > 1) {
       value = [];
-      for (let i = 0, l = options.length; i < l; i++) {
+      const numOfOptions = options.length
```

Comment:
sorry about my previous change request. I haven't notice that options.length was used in the declaration part of the for loop. 
Since the changes are made, you could use `(let i = 0, i < numOfOptions; i++)`
But again, all this "cuisine" is irrelevant if you use a map function